src/core/chatSession.js

```diff
@@ -10,6 +10,7 @@ import { ChatController } from "./chatController";
 import { LogManager, LogLevel, Logger } from "../log";
 import WebSocketManager from "../lib/amazon-connect-websocket-manager";
```

Comment:
lets update version in `package.json` and also update `change.md` so the change details are pickedup in npm history.

Reply:
sure, added a new commit to include changes in `package.json` and `CHANGELOG.md`